src/solvers/dgmulti/dg.jl

```diff
@@ -243,10 +243,25 @@ function max_dt(u, t, mesh::DGMultiMesh,
   # `polydeg+1`. This is because `nnodes(dg)` returns the total number of
   # multi-dimensional nodes for DGMulti solver types, while `nnodes(dg)` returns
   # the number of 1D nodes for `DGSEM` solvers.
-  polydeg = rd.N
   return 2 * dt_min / (polydeg + 1)
 end
 
+function max_dt(u, t, mesh::DGMultiMesh,
+  constant_speed, equations, dg::DGMulti{NDIMS}, cache) where {NDIMS}
+  rd = dg.basis
+  polydeg=Int8
+  if dg.basis.approximation_type isa TensorProductWedge
+    # For tensor-product elements the maximum of the polynomial degrees is used to ensure
+    # that the timestep is feasable for both elements of the tensor-product. For the
+    # other elements taking the maximum has no effect.
+    polydeg=maximum(rd.N)
+  else
+    polydeg = rd.N
+  end
```

Comment:
Why did you do this change? The comment does not match the logic anymore.

Reply:
I changed it after a suggestion of @jlchan. But you're right, the comment needs an update. Nevertheless, the behaviour of max_dt is the same after this change, because Wedges are currently the only elements with a tensor-product-approximation.

Reply:
This should be resolved by the latest suggestion of @jlchan